elections/tac-and-scir-election-process.md

```diff
@@ -0,0 +1,66 @@
+# TAC and SCIR Election Procedures
+
+This document outlines the election process for the OpenSSF Technical Advisory Council (TAC) and Security Community Individual Representative (SCIR).
+
+
+## Voter Eligibility (Electorate) Self-Nomination Process
+
+Any contributor to OpenSSF working groups, SIGs, or projects is eligible to participate in the election.
+Valid contributions include: commits or submitted pull requests via Github; public edits or comments on Google docs or other work products associated with OpenSSF; consistent participation in working groups; posting messages to any mailing list or on Slack; and beyond that any other form of positive engagement with OpenSSF activities.
+The voter eligibility form asks you for an example of your contributions; this is merely to make it easier for the Election Officials and OpenSSF staff to validate eligibility.
+If you have in any way been involved in or care about OpenSSF, but are in doubt as to whether your contribution “counts”, please fill it out anyways, and we will follow up.
+
+[Voter Eligibility Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSegUTOrxrGPVwDrPH9RARmYxX5YEkW3j67RU4uWABkKD5x8og/viewform)
+
+ 
+## TAC Self-Nomination Process
+
+The OpenSSF Technical Advisory Council (TAC) is composed of seven total individuals, four of whom are elected annually.
+Eligible community members, according to the same criteria defined above for voter eligibility, can self-nominate using the form below.
+
+[TAC Candidate Self-Nomination Form](https://docs.google.com/forms/d/e/1FAIpQLSdxUbbfJGgjU_cZ_uGvUm-5ePoXu7q2pGiNyH2G8euwpn23Sw/viewform)
+
+
+## SCIR Self-Nomination Information and Process
+
+Since early in its existence, the OpenSSF Governing Board has sought to ensure it gets adequate input from voices in the software security community who would otherwise not be at the table.
+We seek candidates for the Security Community Individual Representative (SCIR) who can represent those voices, while also being a subject matter expert in the field with their own set of perspectives.
+Familiarity with the different OpenSSF working groups and projects, and being able to dedicate the time to be sufficiently informed on the issues that arise in our monthly calls and ongoing discussions, is highly desired.
+It is also highly desired, but not required, that the SCIR be an eligible OpenSSF community member by the criteria defined above.
```

Comment:
+ @brianbehlendorf to weigh in. I don't think this section accurately captures what we're looking for. SCIR was meant to provide perspectives and insights from Security Experts in communities that we don't typically reach. IIRC, the concern initially was that OpenSSF may not have line of sight into tools, processes, ideas, etc that other security communities develop, and this SCIR role would help us find ways to build bridges across to those worlds.

Reply:
This is fine.